test/routes/signup.test.js

```diff
@@ -0,0 +1,26 @@
+const test = require('tape');
+const supertest = require('supertest');
+const app = require('../../server/app');
+
+test('Test /signup route', (t) => {
```

Comment:
try to add test for someone his username already exists and another for someone his email already exists